.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "deepscan.enable": true
+  "deepscan.enable": true,
+  "liveServer.settings.port": 1987,
```

Comment:
- [ ] Nice review you have here👍👍 Kindly add emoji's to make the review friendlier😉